website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -500,7 +500,7 @@ file. The variables must be set with proper values for your environment in the
 | `db_user_password`       | Password for RDS database user authentication           | "My_Sup3rS3cr3tuserPassw0rd"                                |
 | `dlq_subscription_email` | The email to notify users when messages are received in dead letter SQS queue | "test@email.com"                      |
 | `orca_default_bucket`    | Default ORCA S3 Glacier bucket to use.                  | "PREFIX-orca-primary"                                       |
-| `orca_reports_bucket_arn`| The ARN of the bucket to store s3 inventory reports.    | "arn:aws:s3:::PREFIX-orca-reports"                          |
+| `orca_reports_bucket_name`| The ARN of the bucket to store s3 inventory reports.    | "PREFIX-orca-reports"                          |
```

Comment:
Change `The ARN of the bucket to store s3 inventory reports.` to `The name of the bucket that store s3 inventory reports.`